app/controllers/api/v1/accounts/conversations_controller.rb

```diff
@@ -80,6 +80,12 @@ def toggle_typing_status
     head :ok
   end
 
+  def update_draft_message
+    @conversation.draft_message_content = params.permit(:message)[:message]
```

Comment:
1/ Saving the drafts in a conversation is not ideal. As there are multiple users looking at the same conversation, exposing a draft written by one user to another user is not a great experience. 

2/ The draft message is short-living item, it shouldn't be stored in the database instead it can be saved in Redis storage with key to identify the user, the account and the conversation. The key value pair can be deleted once the message is sent.